cfgov/jinja2/v1/ask-cfpb/category-page.html

```diff
@@ -70,19 +71,27 @@ <h3>What is your question about?</h3>
     <section class="search-results block block__flush-top">
         <div class="search-results_notification u-mt30">
             {% from 'molecules/notification.html' import render as notification with context %}
-            {{ notification('success', false, results_count ~ ' results') }}
+            {{ notification('success', false, results_count ~ _(" results") ) }}
```

Comment:
Will this `_(" results")` work as expected with the leading space?

Reply:
Updated to remove leading space in f06ddeeb84ff023e7b717dc3507e5ec1392e72c0